python/lsst/ts/xml/data/sal_interfaces/MTM1M3/MTM1M3_Commands.xml

```diff
@@ -787,4 +787,37 @@
         Enable all M1M3 force actuators.
     </Description>
   </SALCommand>
+  <SALCommand>
+    <Subsystem>MTM1M3</Subsystem>
+    <EFDB_Topic>MTM1M3_command_enableDisableForceComponent</EFDB_Topic>
+    <Description>
+        Enable and disable various forces components. Can be called only in
+        active engineering mode. Switching from the "active engineering mode"
+        will revert applied flags back to their default values.
+    </Description>
+    <item>
+      <EFDB_Name>forceComponent</EFDB_Name>
+      <Description>Force component to Enable or Disable. One of the EnableDisable_ enumeration member.</Description>
+      <IDL_Type>short</IDL_Type>
+      <Units>unitless</Units>
+      <Count>1</Count>
+      <Enumeration>
```

Comment:
I think enumerations are defined at the top of the [event file](https://github.com/lsst-ts/ts_xml/blob/d5fe4a55c28edb579e156d7d2e89e7b243633f6e/python/lsst/ts/xml/data/sal_interfaces/MTM1M3/MTM1M3_Events.xml#L3), here you should just add something like [this](https://github.com/lsst-ts/ts_xml/blob/d5fe4a55c28edb579e156d7d2e89e7b243633f6e/python/lsst/ts/xml/data/sal_interfaces/MTPtg/MTPtg_Events.xml#L65).

Reply:
Well, that top placement doesn't pass through C++ salobject generator. Looks like a bug in ts_sal then, will verify that.

Reply:
weird, how to you get the enums for DetailedState?

Reply:
I cannot move it to global commands enumeration, as then I lost shared/global event enumerations. Has to keep it that way now.